docs/2.8.1/docs/canton/usermanual/persistence.rst

```diff
@@ -622,8 +622,25 @@ Order of Backups
 
 It is important that the participant's backup is not more recent than that of
 the domain's, as that would constitute a ledger fork. Therefore, if you back up
-both participant and domain databases, always back up the participant database before the
-domain. If you are using a domain integration, then backup the sequencer node before backing
+both participant and domain databases sequentially, the following constraints apply:
+
+- If you run a composite domain node, always back up the participant database before the domain.
+
+- If you run different nodes of the domain separately (sequencer, mediators, domain topology manager):
+
+  - Back up the mediators, the domain manager and participants before the sequencer;
+    otherwise they may not be able to reconnect to the sequencer (``ForkHappened``).
+    The relative order of mediators, domain topology manager, and participants does not matter.
+
+  - On a :ref:`permissioned domain <permissioned-domains>`, make sure that
+    no participant is registered at the domain topology manager
+    between the backup of the domain topology manager and the backup of the sequencer.
+    Otherwise the participant may not work correctly after the recovery.
```

Comment:
```suggestion
  - Back up the mediators, the domain manager, and participants before the sequencer;
    otherwise, they may not be able to reconnect to the sequencer (``ForkHappened``).
    The relative order of mediators, domain topology manager, and participants does not matter.

  - On a :ref:`permissioned domain <permissioned-domains>`, make sure
    no participant is registered with the domain topology manager
    between the backup of the domain topology manager and the backup of the sequencer.
    Otherwise, the participant may not work correctly after the recovery.
```